#!/bin/sh /usr/share/dpatch/dpatch-run
## 206-magic-update-elf-b.dpatch by Robert Millan <rmh@aybabtu.com>
##
## DP: New magic for OpenRISC binary format (Closes: #316076).

@DPATCH@

diff -Naur file-4.20.orig/magic/Magdir/elf file-4.20/magic/Magdir/elf
--- file-4.20.orig/magic/Magdir/elf	2006-12-18 14:14:34.000000000 +0000
+++ file-4.20/magic/Magdir/elf	2007-03-27 16:47:26.000000000 +0000
@@ -99,6 +99,9 @@
 >>18	leshort		62		x86-64,
 >>18	leshort		75		Digital VAX,
 >>18	leshort		88		Renesas M32R,
+>>18	leshort		92		OpenRISC,
+>>18	leshort		0x3426		OpenRISC (obsolete),
+>>18	leshort		0x8472		OpenRISC (obsolete),
 >>18	leshort		94		Tensilica Xtensa,
 >>18	leshort		97		NatSemi 32k,
 >>18	leshort		0x9026		Alpha (unofficial),
@@ -193,6 +196,9 @@
 >>18	beshort		73		Cray NV1,
 >>18	beshort		75		Digital VAX,
 >>18	beshort		88		Renesas M32R,
+>>18	leshort		92		OpenRISC,
+>>18	leshort		0x3426		OpenRISC (obsolete),
+>>18	leshort		0x8472		OpenRISC (obsolete),
 >>18	beshort		94		Tensilica Xtensa,
 >>18	beshort		97		NatSemi 32k,
 >>18	beshort		0x9026		Alpha (unofficial),
